packages/frontend-shared/src/graphql/urqlExchangePubsub.ts

```diff
@@ -7,7 +7,7 @@ export const pubSubExchange = (io: Socket): Exchange => {
     const watchedOperations = new Map<number, Operation>()
     const observedOperations = new Map<number, number>()
 
-    io.on('data-context-push', (...args) => {
+    io.on('data-context-push', () => {
```

Comment:
Any particular reason for removing the `..args` here? I found it useful to notify the client which operation to refetch until the subscriptions land...

In case we want to move forward removing it, should we remove it from here as well? ` toLaunchpad (...args: any[]) {`

Reply:
Nope, we can put this back